src/less/components/topbar.less

```
@@ -197,7 +197,7 @@
                     .nav-group {
                         .nav-ul-active() {
                             > .nav-ul {
-                                max-height: 1500px;
+                                max-height: 1516px;
```

this change seems very hardcoded and therefore is not very futureproof.
It works now that it became broken probably when adding the new dropdown component.
But what if we add 1 or 2 new components again?
This max-height should be "dynamic".
```suggestion
                                max-height: unset;
```
with `unset` value it might fix it in "removing" the `max-height: 0;` set by the other rule on `.nav-ul` ?

---

Yes, I agree.. But the animation won't work without a set max-height.. 😕 Hm, do you have any other suggestions?